schedule/adapter.py

```diff
@@ -26,7 +26,7 @@ def course_to_course_offering(course: Course) -> A_CourseOffering:
 
 
 def add_course_offering_to_schedule(course: Course, a_course_offering: A_CourseOffering):
-    schedule, _ = A_Schedule.objects.get_or_create(id=0)
+    schedule, _ = A_Schedule.objects.get_or_create(id=1)
```

Comment:
I think `get_or_create` is actually not as helpful as intended, because it continues the workflow with an empty Schedule. I think an error case would be better here if `get` fails to return something - "Is your database intialized?" will help out an administrator